tests/zkevm/test_worst_compute.py

```diff
@@ -170,3 +176,167 @@ def test_worst_modexp(
         post={},
         blocks=[Block(txs=[tx])],
     )
+
+
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.parametrize(
+    "gas_limit",
+    [
+        Environment().gas_limit,
+    ],
+)
+@pytest.mark.parametrize(
+    "op",
```

Comment:
A very small improvement, I know, but his helps readability:
```suggestion
    "opcode,opcode_args",
```
See the suggestion added to: https://github.com/ipsilon/execution-spec-tests/pull/8